pkg/util/test/util.go

```diff
@@ -1040,3 +1040,56 @@ func validateStorageClusterIsOnline(cluster *corev1.StorageCluster, timeout, int
 
 	return cluster, nil
 }
+
+// CreatePodSpecWithTLS is a helper method
+func CreatePodSpecWithTLS(CACertFileName, serverCertFileName, serverKeyFileName *string) *corev1.StorageCluster {
```

Comment:
This is creating and returning a StorageCluster spec and not pod spec. Should we call this `CreateClusterWithTLS`?